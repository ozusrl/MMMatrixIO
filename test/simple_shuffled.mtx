%%MatrixMarket matrix coordinate real general
5 5 13
4 3 10.9
3 4 10.6
1 2 10.2
2 3 10.5
1 1 10.1
5 5 11.4
4 5 11.2
4 4 11.1
2 1 10.4
4 2 10.8
5 1 11.3
4 1 10.7
1 5 10.3
