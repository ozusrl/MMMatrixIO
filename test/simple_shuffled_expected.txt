############### MM  ##############
5 5 13
3 2 10.9
2 3 10.6
0 1 10.2
1 2 10.5
0 0 10.1
4 4 11.4
3 4 11.2
3 3 11.1
1 0 10.4
3 1 10.8
4 0 11.3
3 0 10.7
0 4 10.3
############### COO ##############
int rowIndices[13] = {
0,
0,
0,
1,
1,
2,
3,
3,
3,
3,
3,
4,
4,
};
int colIndices[13] = {
0,
1,
4,
0,
2,
3,
0,
1,
2,
3,
4,
0,
4,
};
double values[13] = {
10.1,
10.2,
10.3,
10.4,
10.5,
10.6,
10.7,
10.8,
10.9,
11.1,
11.2,
11.3,
11.4,
};
############### CSR ##############
int rowPtr[6] = {
0,
3,
5,
6,
11,
13,
};
int colIndices[13] = {
0,
1,
4,
0,
2,
3,
0,
1,
2,
3,
4,
0,
4,
};
double values[13] = {
10.1,
10.2,
10.3,
10.4,
10.5,
10.6,
10.7,
10.8,
10.9,
11.1,
11.2,
11.3,
11.4,
};
############### CSC ##############
int rowIndices[13] = {
0,
1,
3,
4,
0,
3,
1,
3,
2,
3,
0,
3,
4,
};
int colPtr[6] = {
0,
4,
6,
8,
10,
13,
};
double values[13] = {
10.1,
10.4,
10.7,
11.3,
10.2,
10.8,
10.5,
10.9,
10.6,
11.1,
10.3,
11.2,
11.4,
};
